### PR TITLE
Fix bug causing remove liquidity to shares to be broken, add withdrawal shares to transaction table

### PR DESCRIPTION
**Pull request by DannyDelott**
Related to #775 

|Desktop|Mobile|
|--|--|
|image|image|

### CODE REVIEW

File: apps/hyperdrive-trading/src/ui/hyperdrive/TransactionTable/TransactionsTable.tsx [hunk 1]
```diff
@@ -123,10 +124,23 @@ function formatTransactionTableMobileData(
...
+  let baseQueuedForWithdrawalLabel;
+  if (
+    row.eventName === "RemoveLiquidity" &&
+    row.withdrawalShares &&
+    row.lpSharePrice
+  ) {
+    const baseQueuedForWithdrawal = dnum.divide(
+      [row.withdrawalShares, hyperdrive.decimals],
+      [row.lpSharePrice, hyperdrive.decimals],
+    )[0];
```
Comment (ryangoree) [inline apps/hyperdrive-trading/src/ui/hyperdrive/TransactionTable/TransactionsTable.tsx:136]: I think this should be `multiply`

File: apps/hyperdrive-trading/src/ui/hyperdrive/TransactionTable/TransactionsTable.tsx [hunk 2]
```diff
@@ -267,6 +286,37 @@ function getColumns(hyperdrive: HyperdriveConfig, appConfig: AppConfig) {
...
+        // show withdrawal shares for remove liquidity
+        if (
+          row.getValue("eventName") === "RemoveLiquidity" &&
+          row.original.withdrawalShares &&
+          row.original.lpSharePrice
+        ) {
+          const baseQueuedForWithdrawal = dnum.divide(
+            [row.original.withdrawalShares, hyperdrive.decimals],
+            [row.original.lpSharePrice, hyperdrive.decimals],
+          )[0];
```
Comment (ryangoree) [inline apps/hyperdrive-trading/src/ui/hyperdrive/TransactionTable/TransactionsTable.tsx:299]: Same here

File: packages/hyperdrive-js-core/src/hyperdrive/ReadHyperdrive/ReadHyperdrive.ts [hunk 3]
```diff
@@ -666,25 +668,19 @@ export class ReadHyperdrive extends ReadModel implements IReadHyperdrive {
     );
   }
   async getLpEvents(): ReturnType<IReadHyperdrive, "getLpEvents"> {
-    const addLiquidtyEvents = await this.contract.getEvents("AddLiquidity");
+    const addLiquidityEvents = await this.contract.getEvents("AddLiquidity");
```
Comment (ryangoree) [inline packages/hyperdrive-js-core/src/hyperdrive/ReadHyperdrive/ReadHyperdrive.ts:671]: Nice catch 🕵️

**Review by ryangoree — APPROVED** [review]

File: apps/hyperdrive-trading/src/ui/hyperdrive/TransactionTable/TransactionsTable.tsx (hunk 1 above)
Comment (DannyDelott) [reply apps/hyperdrive-trading/src/ui/hyperdrive/TransactionTable/TransactionsTable.tsx:136]: d'oh, yes good catch!

**Pull request merged**